Sources/Private/CoreAnimation/Layers/ImageLayer.swift

```diff
@@ -48,6 +48,7 @@ final class ImageLayer: BaseCompositionLayer {
 
     self.imageAsset = imageAsset
     contentsLayer.contents = image
+    contentsLayer.contentsGravity = context.imageProvider.contentsGravity(for: imageAsset) ?? .resize
```

Comment:
We should also make this change to the `ImageCompositionLayer` used by the Main Thread rendering engine